OpenUpSA/wazimap-ng-ui
# overlapping marker spiderfier

## Description
making it easier for the users to click on markers that overlap with each other

## Related Issue


## How to test it locally
* select a theme from the service mapper
* wait until the markers are created
* click on a group of markers
* confirm that the marker that is clicked is now separated from the others

## Screenshots
![image](https://user-images.githubusercontent.com/53019884/117676635-65803f80-b1b6-11eb-8e03-518c92512d64.png)
![image](https://user-images.githubusercontent.com/53019884/117676645-687b3000-b1b6-11eb-8eba-2d062a1c3987.png)


## Changelog

### Added

### Updated
* `point_data.js` to use `overlapping-marker-spiderfier-leaflet` library

### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
